courts_db/data/courts.json

```diff
@@ -29168,8 +29438,13 @@
             "Surrrogate.* New York",
             "Snrrogate S Court of New York New York County",
             "New York Surrogate's Court, Ulster County",
-            "Surrogate's Court, New York County",
-            "New York Surrogate's Court, \b.*\b County"
+            "Surrogate'?s? Court(\\.|,)? (Albany|Allegany|Broome|Bronx|Cattaraugus|Cayuga|Chautauqua|Chemung|Chenango|Clinton|Columbia|Cortland|Delaware|Dutchess|Erie|Essex|Franklin|Fulton|Genesee|Greene|Hamilton|Herkimer|Jefferson|King|Kings|Lewis|Livingston|Madison|Montgomery|Monroe|Nassau|Niagara|New York|Orleans|Oneida|Onondaga|Ontario|Orange|Otsego|Oswego|Putnam|Queens|Rennsselaer|Rensselaer|Richmond|Rockland|Saratoga|Schenectady|Schoharie|Schuyler|Seneca|St. Lawrence|Steuben|Suffolk|Sullivan|Tioga|Tompkins|Ulster|Warren|Washington|Wayne|Westchester|Wyoming|Yates) County",
+            "New York Surrogate's Court, \b.*\b County",
+            "Surrogate's Court",
+            "Surrogates' Court, New York County",
+            "Surrogate'? Court, (Allegany|Cattaraugus|Kings|Saratoga|Westchester) County",
+            "Surrogate's Court of the City of New York(\\.|,)? (Albany|Bronx|Chautauqua|Dutchess|Erie|Kings|Nassau|Monroe|New York|Queens|Richmond|Rockland|Schenectady|Schoharie|Schuyler|Seneca|St. Lawrence|Steuben|Suffolk|Westchester|Wyoming) County",
+            "Surrogate's Court of the State of New York, Nassau County"
```

Comment:
this would be cleaner if it was just broken up into each county - surrogate courts. I think.  

so parent Ny surrogate court and then each subdivision no?